configuration/configuration.go

```diff
@@ -562,6 +573,13 @@ func assertConstructionConfiguration(config *ConstructionConfiguration) error {
 		return fmt.Errorf("%w: invalid value for MaximumFee", err)
 	}
 
+	for _, account := range config.PrefundedAccounts {
```

Comment:
We should also make sure it is a valid curve type and that the address is greater than length 0...I think there is an `asserter` method for this.